pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -534,6 +535,10 @@ buildStdenv.mkDerivation ({
     }
   '';
 
+  postFixup = lib.optionalString crashreporterSupport ''
+    patchelf --add-rpath "${lib.makeLibraryPath [ curl ]}" $out/lib/${binaryName}/crashreporter
```

Comment:
Reuses `binaryName` now, I guess this is what broke non-firefox consumers.

Reply:
I confirm that `thunderbird-unwrapped` builds now and its `crashreporter` binary looks OK.